src/docs/truffle/getting-started/running-migrations.md

```diff
@@ -113,34 +113,40 @@ Filename: `migrations/1_initial_migration.js`
 ```javascript
 var Migrations = artifacts.require("Migrations");
 
-module.exports = function(deployer) {
-  // Deploy the Migrations contract as our only task
-  deployer.deploy(Migrations);
+module.exports = function(deployer, networks, accounts) {
+  deployer.then(async () => {
+    // Deploy the Migrations contract as our only task
+    await deployer.deploy(Migrations);
+  });
 };
 ```
 
 From here, you can create new migrations with increasing numbered prefixes to deploy other contracts and perform further deployment steps.
 
 ## Deployer
 
-Your migration files will use the deployer to stage deployment tasks. As such, you can write deployment tasks synchronously and they'll be executed in the correct order:
+Your migration files will use the deployer to stage deployment tasks. As such, you can write deployment tasks asynchronously:
 
 ```javascript
 // Stage deploying A before B
 deployer.deploy(A);
 deployer.deploy(B);
 ```
 
-Alternatively, each function on the deployer can be used as a Promise, to queue up deployment tasks that depend on the execution of the previous task:
+Alternatively, use async to deploy queue up deployment tasks that depend on the execution of the previous task:
 
 ```javascript
-// Deploy A, then deploy B, passing in A's newly deployed address
-deployer.deploy(A).then(function() {
-  return deployer.deploy(B, A.address);
-});
+module.exports = function(deployer, networks, accounts /* See note 1 */) {
+  deployer.then(async () => {
```

Comment:
Same as the above issue. You can just basically take out line 140 and 143 and it will be correct.